CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+[DOC]     Whenever you do something only(!) related to the documentation
+[INFRA]   Whenever you change something of the build system or CI related
+[TEST]    Whenever you do something related to the tests (unit or benchmark)
+[MISC]    Whenever it does not fit to any of the above
+```
+
+E.g. `[FEATURE] add support for FOO` or `[FIX] divide by zero in alignment`.
+
+**Alternatively, if your changes are so small that they would fit into one commit, don't worry about any of the above,
+just state "Please stash commits" in the PR and we will take care of the rest.**
+
+## Opening a pull-request ("PR")
+
+When you are done with committing changes to your branch and you have tested your changes, open a pull request.
+We have continuous integration in place that should inform you of test failures. Please try to resolves any
+breakage that your pull request introduces.
+If there are test failures that you don't understand clearly indicate that you have seen the errors, but cannot resolve
+them – then the first reviewer will have a look at them. Otherwise the PR will be treated as still being
+work-in-progress.
+
+## The review process
+
+### Request first review
+
+After opening a PR it goes through the review process. This is a two-step process, first a regular member of the team
+needs to approve your changes, then one of the project owners (`@h-2` or `@rrahn`) needs to approve and merge it. If you
+know the SeqAn team member best suitable for the review or GitHub suggests exactly one person, you can request a review
+from that person. Otherwise wait for a reviewer to be assigned to the PR. Do not request reviews from a project owner
+at this point. Always only request review from one person at a time.
+
+### Resolve comments
+
+If you have not contributed to SeqAn before, you will likely receive a lot of comments on the style of your code. This
+is not a sign that we don't appreciate your contribution, but we have high code quality standards and everything
+must conform to our [style guide](https://github.com/seqan/seqan3/wiki#library-coding-guide). In the future we hope
+to automate this step.
+After you have received the first review, go through all comments and clarify any suggestions and requests. When
+you are confident that you know how to address all comments, implement the changes and update your PR.
+
+### Re-request review
+
+Now re-request a review from the previous reviewer. **This step is important**, otherwise your PR is treated as still
+being work-in-progress! The reviewer will go through their first review and mark all previous comments as resolved if
+they think you resolved them (do not mark another person's comments as resolved!). They will then add a
+second review or approve your changes. Repeat the last steps until you get approval.
+
+### Request project owner review
+
+Once your changes are approved by one reviewer, request a second review from a project owner. They will go through
```

Comment:
It's mentioned above and I didn't want to have it in too many places so the chances are smaller to become inconsistent when changed.

Reply:
Mhm, I thought it would make sense to have the name of the owners in the part where you need to assign them.
But I guess the guide is short enough to find it easily.